library/Zend/Pdf.php

```diff
@@ -576,6 +590,77 @@ protected function _loadOutlines(Zend_Pdf_Element_Reference $root)
     }
 
     /**
+     * Load form fields
+     * Populates the _formFields array, for later lookup of fields by name
+     *
+     * @param Zend_Pdf_Element_Reference $root Document catalog entry
+     */
+    protected function _loadFormFields(Zend_Pdf_Element_Reference $root)
+    {
+        if ($root->AcroForm === null || $root->AcroForm->Fields === null) {
+            return;
+        }
+        
+        foreach ($root->AcroForm->Fields->items as $field)
+        {
+            /* We only support fields that are textfields and have a name */
+            if ( $field->FT && $field->FT->value == 'Tx' && && $field->T && $field->T !== null ) 
+            {
+                $this->_formFields[$field->T->value] = $field;
+            }
+        }
+		
+        if ( !$root->AcroForm->NeedAppearances || !$root->AcroForm->NeedAppearances->value )
+        {
+            /* Ask the .pdf viewer to generate its own appearance data, so we do not have to */
+            $root->AcroForm->add(new Zend_Pdf_Element_Name('NeedAppearances'), new Zend_Pdf_Element_Boolean(true) );
+            $root->AcroForm->touch();
+        }    
+    }
+	
+    /**
+     * Retrieves a list with the names of the AcroForm textfields in the PDF
+     *
+     * @return array of strings
+     */
+    public function getTextFieldNames()
+    {
+        return array_keys($this->_formFields);
+    }
+	
+    /**
+     * Sets the value of an AcroForm text field
+     *
+     * @param string $name Name of textfield
+     * @param string $value Value
+     * @throws Zend_Pdf_Exception if the textfield does not exist in the pdf
+     */
+    public function setTextField($name, $value)
+    {
+        if ( !isset($this->_formFields[$name]))
```

Comment:
I created an issue here, I hope that is the right place: https://github.com/zendframework/ZendPdf/issues/22